git.md

```diff
@@ -198,9 +198,41 @@ Where relevant, go through the changes in deploy preview or in your local enviro
 
 Prioritize on reviewing those that marked as bug fixes, then the new features. Use `G2M` as a search filter to minimize noise -- provided that it is a well-followed convention in the given project.
 
+## On Github Projects
+
+Since the [Covid Dash development](https://github.com/EQWorks/flaat-dash/projects), we have been using github projects to keep track of progress, issues, and code-specific discussions.
+
+Repos can have multiple projects. Projects can be specific to one extensive task or the whole repo in general - ideally both.
+Projects are basically a to-do list easy to keep track of progress as it shows the percentage between open issues / in progress issues / completed issues.
+
+#### On issues
+An issue is created as a to-do item, assigned to a person and labeled:
+- On the ToDo card, click `+`, add a title to your issue, and click `add` a note.
+- Once the note is created, click on `...` of that new ToDo note and choose `convert to issue`.
+- Click on the title of that issue now and a side card will pop on the right. Choose an assignee, add a description, add labels - multiple labels can be added according to the context of that issue.
+- Move the issue between cards to update its status accordingly: when assigned to someone, move to `in progress`, when PR is up, move to `Review in progress`
+
+#### On Labels
+most common uses are:
+- bug - of a feature that already exists
+- layout/design - style modifications
+- enhancement - improving an existing feature
+- feature - creating a new functionality
+
+#### On PR
```

Comment:
try to correlate this with the existing [pull/merge request section](https://github.com/EQWorks/common/blob/master/git.md#on-pullmerge-requests).